client/src/actions/userActions.js

```diff
@@ -71,3 +72,21 @@ export function checkIfLoginIsValid() {
     failAction: logoutUser
   });
 }
+
+export function sendPasswordReset(formData) {
+  return createApiAction({
+    callApi: () => post('users/reset', {body: formData}),
```

Comment:
It looks like `eslint` is requesting spaces in this object (see: https://travis-ci.org/codefordenver/encorelink/builds/203988812). You can run eslint locally with `npm run lint`, and I would highly recommend making sure your editor shows you inline errors.